command/genesis/params.go

```diff
@@ -61,7 +61,8 @@ var (
 	errInvalidEpochSize       = errors.New("epoch size must be greater than 1")
 	errInvalidTokenParams     = errors.New("native token params were not submitted in proper format " +
 		"(<name:symbol:decimals count:mintable flag:[mintable token owner address]>)")
-	errRewardWalletAmountZero = errors.New("reward wallet amount can not be zero or negative")
+	errRewardWalletAmountZero  = errors.New("reward wallet amount can not be zero or negative")
+	errEscrowAccMustBePremined = errors.New("it is mandatory to premine escrow account (0x0 address)")
```

Comment:
Why you calling it `escrow` account?

Reply:
IDK, sounds a bit off to me, that definition resembles to me as the owner of the tokens, more like a `reserve`, wdyt?

Reply:
`tokenReserve`, `reserve`, `treasury`, something like that.